src/test/resources/features/locations.feature

```diff
@@ -25,10 +25,6 @@ Feature: Locations
     When a request is made to retrieve location with locationId of "-9999"
     Then resource not found response is received from locations API
 
-  Scenario: Request for list/group that does not exist
```

Comment:
this wasn't testing the endpoint as it didn't exist, more of a test for the global not found handler which is not required.